es_gui/apps/data_manager/widgets.py

```diff
@@ -456,21 +456,348 @@ def execute_download(self):
 
             self.n_active_threads = len(job_batches)
 
+            five_minute_start = datetime.datetime(2017, 12, 1)
+            
             # (Re)set the progress bar and output log.
             self.progress_bar.value = 0
-            self.progress_bar.max = total_months*total_nodes + total_months
+            self.progress_bar.max = 2*total_months
             self.output_log.text = ''
 
             # Check connection settings.
             ssl_verify, proxy_settings = check_connection_settings()
 
             # Spawn a new thread for each download_ISONE_data call.
             for batch in job_batches:
-                thread_downloader = threading.Thread(target=self._download_ISONE_data, args=(acc_user, acc_pw, batch[0], batch[-1]),
+                thread_downloader = threading.Thread(target=self.download_ISONE_data_updated, args=(acc_user, acc_pw, batch[0], batch[-1]),
                                                      kwargs={'ssl_verify': ssl_verify, 'proxy_settings': proxy_settings, 'nodes':nodes_selected})
 
                 thread_downloader.start()
 
+    #//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////#
+    def download_ISONE_data_updated(self, username, password, datetime_start, datetime_end=None, nodes=[], typedat="all", path='data/', ssl_verify=True, proxy_settings={}):
```

Comment:
If the functionality of this method replaces the old method, we should replace it instead of creating a new one.